### PR TITLE
fantia.jp

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/108651

<details><summary>Screenshot:</summary>

Снимок экрана 2022-02-21 в 12 23 10

Снимок экрана 2022-02-21 в 12 23 40

</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -126,6 +126,7 @@ xn----7sba7aachdbqfnhtigrl.xn--j1amh#$#html { overflow: visible!important; }
 backblaze.com##.ju_Con
 honeysanime.com##.buruburu-bee
 nhac.vn##.ft-scroll-content
+fantia.jp##.fanclub-btns > div.popup-bar + div.popup-bar
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:129]: `fantia.jp##.fanclub-btns > div.popup-bar:last-child` or `fantia.jp##.fanclub-btns > div.popup-bar.is-clone`.

File: AnnoyancesFilter/sections/self-promo.txt [hunk 2]
```diff
@@ -1259,6 +1259,7 @@ zerohedge.com##div[class^="TheMarketEarHomePageSidebar_container__"]
 cyberleninka.ru#?#.infoblock:has(> div[class="full"] > div.bibloid_serp)
 drwindows.de#?#section[class^="artikel-"] > ul > li:not([class]):has(> a[href] > h3.post:contains(Anzeige))
 stopgame.ru##.corner_banner
+fantia.jp###post-footer
```
Comment (zloyden) [inline AnnoyancesFilter/sections/self-promo.txt:1262]: I don't think we should block the whole element entirely, just `share` part. @Yuki2718 please also take a look.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/self-promo.txt (hunk 2 above)
Comment (Yuki2718) [reply AnnoyancesFilter/sections/self-promo.txt:1262]: I agree, either extended syntax or `###post-footer > div.well-overlay-white:last-child`

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (Yuki2718) [reply AnnoyancesFilter/sections/popups.txt:129]: Personally I prefer the latter.

**Review by Yuki2718 — APPROVED** [review]

File: AnnoyancesFilter/sections/self-promo.txt [hunk 3]
```diff
@@ -1259,6 +1259,7 @@ zerohedge.com##div[class^="TheMarketEarHomePageSidebar_container__"]
 cyberleninka.ru#?#.infoblock:has(> div[class="full"] > div.bibloid_serp)
 drwindows.de#?#section[class^="artikel-"] > ul > li:not([class]):has(> a[href] > h3.post:contains(Anzeige))
 stopgame.ru##.corner_banner
+fantia.jp###post-footer > div.well-overlay-white:last-child
```
Comment (zloyden) [inline AnnoyancesFilter/sections/self-promo.txt:1262]: Wrong file - it's share.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**